test/certs/domain.cfg

```diff
@@ -0,0 +1,30 @@
+# X.509 Certificate options
+organization = "Koko inc."
+unit = "sleeping dept."
+locality = "foobar"
+state = "Attiki"
+country = GR
+cn = "Cindy Lauper"
+uid = "clauper"
```

Comment:
I'm a little uncomfortable with using the name of real-world individuals. Otherwise LGTM, thank you for taking this on so quickly.

Reply:
Lol, I literally copy-pasted that out of the man page.  But you're right, we should use your name instead :grin:

Reply:
thx for looking.